### PR TITLE
PUBLISH IMMEDIATELY PLEASE. fix decapoids having all access

### PR DESCRIPTION
**Pull request by widgetbeck**
**Changelog**

:cl:
- fix: Fixed Decapoids being able to pry powered doors.

### CODE REVIEW

**Comment by Darkmajia** [comment]
oopsie

File: Resources/Prototypes/Loadouts/Miscellaneous/survival.yml [hunk 1]
```diff
@@ -62,7 +62,6 @@
   storage:
     lhand:
     - BoxSurvival
-
```
Comment (Darkmajia) [inline Resources/Prototypes/Loadouts/Miscellaneous/survival.yml:65]: why do you hate this line break so much.. this is the second time!!!!

**Comment by widgetbeck** [comment]
accident

**Review by hivehum — APPROVED** [review]
funny edit. sorry

**Pull request merged**